src/impact-dashboard/ImpactProjectionTable.tsx

```diff
@@ -0,0 +1,162 @@
+import numeral from "numeral";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+
+export interface TableRow {
+  label: string;
+  week1: number | null;
+  week2: number | null;
+  week3: number | null;
+  overall: number | null; // n/a for some rows
+}
+
+interface Props {
+  incarceratedData: TableRow[];
+  peakData: TableRow[];
+  staffData: TableRow[];
+}
+
+const Table = styled.table`
+  color: ${Colors.black};
+  font-family: "Rubik", sans-serif;
+  font-size: 12px;
+  font-weight: normal;
+  letter-spacing: 0;
+  text-align: center;
+  width: 100%;
+  max-width: 600px;
+
+  th,
+  td {
+    padding-bottom: 5px;
+  }
+`;
+
+const TableHeading = styled.thead`
+  border-bottom: 1px solid ${Colors.forest};
+`;
+
+const HeadingCell = styled.th<{ left?: boolean }>`
+  color: ${Colors.forest};
+  font-family: "Poppins", sans-serif;
+  font-size: 14px;
+  font-weight: 600;
+  letter-spacing: -0.05em;
+  text-align: ${(props) => (props.left ? "left" : "center")};
+
+  &.marked {
+    position: relative;
+    &::after {
+      content: "";
+      position: absolute;
+      left: calc(50% - 0.5px);
+      bottom: -4px;
+      width: 0;
+      height: 9px;
+      border-color: ${Colors.forest};
+      border-left: 1px solid;
+      overflow: visible;
+    }
+  }
+`;
+
+const TableSection = styled.tbody`
+  &::before {
+    content: "";
+    display: table-row;
+    height: 16px;
+  }
+`;
+
+const LabelCell = styled.td<{ italic?: boolean }>`
+  color: ${Colors.forest};
+  font-weight: normal;
+  text-align: left;
+  opacity: 0.8;
+  font-style: ${(props) => (props.italic ? "italic" : "inherit")};
+`;
+
+numeral.nullFormat("N/A");
```

Comment:
Whoa, this seems kind of global. If code somewhere else calls `numeral.nullFormat("something else")`, then the behavior of the code below is nondeterministic.

I think the most proper way to do this is to move these formatting functions into a separate file, so it's clear that that's the only place `numeral` functions are being run.

Otherwise, we can just add a TODO here for now.

Reply:
yeah it does, doesn't it .... I have a workaround that is probably safer, i'll just replace this